tests/page_objects/project_search_page.py

```
@@ -0,0 +1,60 @@
+        
+        search_button = self.driver.find_element(*ProjectSearchLocators.SEARCH_BUTTON)
+        search_button.click()
+
+    def is_project_displayed(self):
+        """Check if project information is displayed."""
+        try:
+            self.wait.until(
+                EC.presence_of_element_located(ProjectSearchLocators.PROJECT_RESULT)
+            )
+            return True
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)